youtube_dl/extractor/common.py

```diff
@@ -43,6 +43,7 @@
     GeoRestrictedError,
     GeoUtils,
     int_or_none,
+    orderedSet,
```

Comment:
Sorry I may be inaccurate. Capitalized symbols are sorted together with non-capitalized symbols. For example, this is correct:
```
     error_to_compat_str,
     ExtractorError,
     fix_xml_ampersands,
     float_or_none,
     GeoRestrictedError,
```